src/arc_utilities/filesystem_utils.py

```diff
@@ -35,3 +39,28 @@ def directory_size(dir: pathlib.Path):
 
 def append_str_to_path(p: pathlib.Path, s: str):
     return p.parent / (p.name + s)
+
+
+def ask_to_remove_directories(directories_to_remove: List[pathlib.Path]):
+    print("Ok to delete these directories?")
+    for d in directories_to_remove:
+        print(d.as_posix())
+    k = input("[Y/n]")
+    if k == 'n' or k == 'N':
+        print(Fore.RED + "Aborting.")
+        return
+
+    print(Fore.GREEN + "Deleting.")
```

Comment:
This logic is deceptive; you abort only if you input `n` or `N`, while anything else would lead to delete. Is this consistent with other cases of `[Y/n]`? I thought it meant yes only if you input nothing, `y`,  or `Y`? For anything else it should just ask again.

Reply:
ok I'll fix this, I just did this because it was simpler